### ISSUE TITLE
Проблемы с логикой.

### ISSUE THREAD

**Issue by yaroslav-hub** [chkqq/TravelLine#5]
Нарушена логика программы. Почему функция проверки на корректность что-то делает помимо проверки на корректность?

https://github.com/chkqq/TravelLine/blob/ba64999f9adc59dbbd48955d89da3d34331956f7/FIO/FIO/Program.cs#L12

**Comment by chkqq** [comment]
Я её написал так, чтобы она проверяла на корректность сразу какую-то
переменную, и выводила сообщение об ошибке в консоль. То есть нужно убрать
вывод в консоль?

чт, 27 июл. 2023 г., 15:54 Yaroslav ***@***.***>:

**Comment by yaroslav-hub** [comment]
Верно. Поведение метода должно соответствовать именованию. Метод IsValid() подразумевает проверку чего-то на валидность и ответ bool.
